### ISSUE TITLE
How put front server in /app folder for url

### ISSUE THREAD

**Issue by kfranco89** [STS-Rosario/carpoolear#47]
Hi i have problem with url , i create a sub domain for backend file but send me problems with CORS so could you explain how create structure like http://localhost/app  for front server and mantain http://localhost for back server i use ubuntu..  

thanksyou

**Comment by p4bl1t0** [comment]
You have to push the frontend web build code into the public folder of the backend. You can make a symbolic link

**Comment by kfranco89** [comment]
By symbolic link, do you mean virtual host?

**Comment by p4bl1t0** [comment]
No, I meant symbolic link.
https://www.shellhacks.com/symlink-create-symbolic-link-linux/

**Comment by kfranco89** [comment]
so should be  ln -s /home/Desktop/carpoolear /Home/Desktop/carpoolear_backend/public/app 

or ?

**Comment by p4bl1t0** [comment]
Yes something like that. Assuming that in /home/Desktop/carpoolear is the build of the front-end project

**Comment by raditha** [comment]
@kfranco did you resolve this issue?
I am getting a blank page when the dist content is put to the public/app path and php server is deployed. npm run dev works fine.

**Comment by p4bl1t0** [comment]
You have to use  the cordova browser build folder in order to work.

**Comment by raditha** [comment]
> You have to use the cordova browser build folder in order to work.

How do we host the backend in apace2 along with the frontend? The frontend loads properly but the backend returns a 404 for /api/ calls.
